src/components/OperationRow.js

```diff
@@ -56,6 +56,7 @@ class OperationRow extends PureComponent<Props, *> {
               ellipsizeMode="tail"
               style={[styles.topLine, { color: valueColor }]}
             >
+              {isIn ? "+ " : "- "}
```

Comment:
This shouldn't be done like this, in fact it's part of formatCurrencyUnit to do this (there is utilities in live-common to have the actual value from the operation + a special mode to force display of "+")

Idea of this is that potentially the way this is formatted depends on the locale

Reply:
ok I need to check this out then